### PR TITLE
add test for domain pillow + misc cleanup

### PR DESCRIPTION
**Pull request by czue**
no functional changes besides the new test. just wanted to get this out of the way before other stuff increased the scope.

@snopoke / cc @nickpell 

wait for tests.

### CODE REVIEW

File: testapps/test_pillowtop/tests/test_reindexer.py [hunk 1]
```diff
@@ -36,6 +36,7 @@ def test_domain_reindexer(self):
             ('ptop_reindexer_v2', {'index': 'domain', 'cleanup': True, 'noinput': True})
         ]:
             delete_all_domains()
+            ensure_index_deleted(DOMAIN_INDEX)
```
Comment (czue) [inline testapps/test_pillowtop/tests/test_reindexer.py]: @millerdev I think this should fix one of the test issues you mentioned

**Comment by snopoke** [comment]
Test failure:
```
======================================================================
FAIL: test_domain_reindexer (testapps.test_pillowtop.tests.test_reindexer.PillowtopReindexerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/testapps/test_pillowtop/tests/test_reindexer.py", line 44, in test_domain_reindexer
    self.assertEqual(1, results.total)
AssertionError: 1 != 2
```

**Comment by millerdev** [comment]
test failure
```
FAIL: test_basic_devicelog (couchforms.tests.test_devicelogs.DeviceLogTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/corehq/util/test_utils.py", line 228, in inner
    return helper(*args, **kwargs)
  File "/mnt/corehq/util/test_utils.py", line 201, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/mnt/corehq/util/test_utils.py", line 216, in call_with_settings
    fn(*args, **kwargs)
  File "/mnt/corehq/util/test_utils.py", line 197, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "corehq/ex-submodules/couchforms/tests/test_devicelogs.py", line 40, in test_basic_devicelog
    self.assert_force_close_entries()
  File "corehq/ex-submodules/couchforms/tests/test_devicelogs.py", line 104, in assert_force_close_entries
    self.assertEqual(force_closure.date.isoformat(), '2016-03-15T13:37:04.573000')
AssertionError: '2016-03-15T07:52:04.573000' != '2016-03-15T13:37:04.573000'
```

**Comment by czue** [comment]
thanks - i think all builds are failing with that error, but good to wait till it's fixed just in case

**Comment by czue** [comment]
:white_check_mark:

**Pull request merged**